x11rb-protocol/src/utils.rs

```diff
@@ -59,20 +51,19 @@ mod raw_fd_container {
         /// This is similar to dropping the `RawFdContainer`, but it allows
         /// the caller to handle errors.
         pub fn close(self) -> Result<(), std::io::Error> {
-            let fd = self.into_raw_fd();
-            nix::unistd::close(fd).map_err(|e| e.into())
+            todo!()
         }
     }
 
-    impl<T: IntoRawFd> From<T> for RawFdContainer {
```

Comment:
Perhaps at least add an impl for `From<OwnedFd>` (yes, `new()` already does exactly that)?

`FromRawFd` would be nice to have, but that is `unsafe` and thus causes problems, I guess.